pulp_ansible/app/migrations/0051_force_index_build.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.postgres import search as psql_search
+from django.db import migrations
+
+from pulp_ansible.app.tasks.collectionversion_index import rebuild_index
+
+
+def force_build(*args, **kwargs):
+    rebuild_index()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ansible', '0050_crossrepositorycollectionversionindex'),
```

Comment:
We haven't released this feature yet, I would just add this to the original migration.